modules/contact/test/tests/bouncing-block-contact/bouncing-block-ranfs-action.i

```diff
@@ -4,11 +4,10 @@ offset = 1e-2
 [GlobalParams]
   displacements = 'disp_x disp_y'
   diffusivity = 1e0
-  ping_pong_protection = true
```

Comment:
There are a few changes to this file beyond just adding the Contact action.  I'd leave these parameters as they were.  Also, I would not re-name the file.

Reply:
Thanks Ben! Please see the updated files which should address your previous comments. I added the 'ping_pong_protection' in the contact action while preventing other contact formulations from trying to use it. It works fine, but I am not sure if this would be the best code design. Let me know.

Reply:
Looks great. Thanks!